src/Metrics.ts

```diff
@@ -7,33 +7,58 @@ const log = new Logger("Metrics");
 export class Metrics {
     public readonly expressRouter = Router();
 
-    public readonly webhooksHttpRequest = new Counter({ name: "hookshot_webhooks_http_request", help: "Number of requests made to the hookshot webhooks handler", labelNames: ["path", "method"], registers: [this.registry]});
```

Comment:
N.B. I was attempting to fix https://github.com/matrix-org/matrix-hookshot/issues/729, but gave up when the problem was more widespread. This *is* still an improvement though, so keeping it in.